workers/unity/Packages/com.improbable.gdk.core/Systems/CustomSpatialOSSendSystem.cs

```diff
@@ -22,7 +23,10 @@ protected override void OnCreateManager(int capacity)
             worker = WorkerRegistry.GetWorkerForWorld(World);
 
             spatialOSSendSystem = World.GetOrCreateManager<SpatialOSSendSystem>();
-            if (!spatialOSSendSystem.TryRegisterCustomReplicationSystem(typeof(T)))
+
+            var component = Activator.CreateInstance(typeof(T)) as ISpatialComponentData;
```

Comment:
This can be `new T()`

Reply:
I think you'll need a `new()` type constraint as well, if memory serves.

Reply:
That's probably okay, I think structs satisfy that constraint by default